torch/ao/quantization/fx/_convert_do_not_use.py

```diff
@@ -125,6 +125,26 @@ def run_weight_observers(observed: GraphModule) -> None:
             # run the weight observer
             weight_observer_module()
 
+# this method is temporary will be removed soon
```

Comment:
Is the real fix removing the check for whether dequantize nodes have multiple users in the lowering code? Is there a reason why we don't just directly change that instead of adding a temporary fix?